doc/examples/template.json

```diff
@@ -7,17 +7,24 @@
 	 * to set the timer's period to 0
 	 */
 	"tasks" : {
+		/* no 'loop' value at task level means loop indefinitely (-1) */
 		"thread0" : {
-			"instance" : 1,
-			"loop" : -1,
-			"run" :   10000,
-			"sleep" : 0,
+			"instance" : 1,  /* number of threads to be created */
+			"loop" : -1,  /* loop forever (see 'duration' below) */
+
+			"run" :   10000,  /* when woken up,
+				execute 10000 / pLoad loops (note: rounding) */
```

Comment:
the comment is confusing. /* when woken up, run for 10000 usec. The duration is converted in a number of 10000/pload loops ( floor rounded) */

Reply:
i thought it's clearer the way i wrote it (because you don't actually run for the exact amount of time) but then i'm biased.